### ISSUE TITLE
CheckBox: Multiple keros scan failures

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#5058]
Keros for chrome scan on CheckBox component is giving multiple errors.

Scanned component: https://developer.microsoft.com/en-us/fabric#/components/checkbox
![image](https://user-images.githubusercontent.com/11937034/40826106-524b2b46-6597-11e8-944a-23b3b0af3b92.png)

#### Actual behavior:

The following rules fail:

- keros-redundant-labels-instructions
- keros-redundant-cues
- aria-valid-attr-value

![image](https://user-images.githubusercontent.com/11937034/40829773-1e965e82-65a2-11e8-9d05-bddd9f5cbed0.png)

![image](https://user-images.githubusercontent.com/11937034/40829795-2f535806-65a2-11e8-8997-3f7d051519f5.png)

#### Expected behavior:

Keros scan should not give any errors

**Comment by JasonGore** [comment]
This issue was fixed at least in part by #5040. I have also run Keros on current Checkbox and currently no longer see any errors related to the Checkbox examples. I will close this issue for now, but please feel free to reopen it if you are still seeing any of the errors you reported. Thanks.
